src/meili/geometry_helpers.cc

```diff
@@ -0,0 +1,101 @@
+#include "meili/geometry_helpers.h"
```

Comment:
i like that you made this a source file but it makes it impossible to tell exactly what was changed. is there any way you could prepare a diff so that its easier for reviewers to tell what the actual changes were?

Reply:
I hear ya... what is the best way to do that?

Reply:
locally you could paste the body of the method back into the header and then do something like `git diff origin/master -- valhalla/meili/geometry_helpers.h` and then post that diff here. that should show just the changes in the header wrt to what is in master (i think, this is off the top of my head)